src/mlpack/methods/ann/layer/dropout.hpp

```diff
@@ -47,21 +47,17 @@ namespace ann /** Artificial Neural Network. */ {
  * @tparam OutputDataType Type of the output data (arma::colvec, arma::mat,
  *         arma::sp_mat or arma::cube).
  */
-template <
-    typename InputDataType = arma::mat,
-    typename OutputDataType = arma::mat
->
+template<typename InputDataType = arma::mat,
+    typename OutputDataType = arma::mat>
```

Comment:
I don't see any need to change from before, but if you do change the style here, can you line up the `typename OutputDataType` with the `typename InputDataType` please?

Reply:
Done :+1: